src/test/deserialization_tests.cpp

```diff
@@ -202,14 +215,25 @@ TEST_P(GRPCPredictRequestNegative, ShouldReturnDeserializationErrorForSetBlobExc
     EXPECT_EQ(status, ovms::StatusCode::OV_INTERNAL_DESERIALIZATION_ERROR) << status.string();
 }
 
-TEST_P(GRPCPredictRequest, ShouldSuccessForSupportedPrecision) {
-    Precision testedPrecision = GetParam();
-    tensorMap[tensorName]->setPrecision(testedPrecision);
+TEST_F(GRPCPredictRequest, ShouldSuccessForSupportedPrecision) {
+    InferenceEngine::TensorDesc tensorDesc(Precision::FP32, shape_t{1, 10}, InferenceEngine::Layout::NC);
+    std::shared_ptr<ovms::TensorInfo> tensorInfo = std::make_shared<ovms::TensorInfo>(
+        std::string("b"),
+        tensorDesc.getPrecision(),
+        tensorDesc.getDims(),
+        tensorDesc.getLayout());
+    tensorMap[tensorName] = tensorInfo;
     std::shared_ptr<MockIInferRequest> mInferRequestPtr = std::make_shared<MockIInferRequest>();
-    InferenceEngine::InferRequest inferRequest(mInferRequestPtr);
-    EXPECT_CALL(*mInferRequestPtr, SetBlob(_, _, _));
+    InferenceEngine::Core engine;
+    InferenceEngine::CNNNetwork network = engine.ReadNetwork(std::filesystem::current_path().u8string() + "/src/test/dummy/1/dummy.xml");
+    InferenceEngine::ExecutableNetwork execNetwork = engine.LoadNetwork(network, "CPU");
+    InferenceEngine::InferRequest inferRequest = execNetwork.CreateInferRequest();
+    std::shared_ptr<NiceMock<MockBlob>> mockBlobPtr = std::make_shared<NiceMock<MockBlob>>(tensorDesc);
+    inferRequest.SetBlob("b", mockBlobPtr);
     InputSink<InferRequest&> inputSink(inferRequest);
     auto status = deserializePredictRequest<ConcreteTensorProtoDeserializator>(request, tensorMap, inputSink, isPipeline);
+    SPDLOG_ERROR(status.getCode());
```

Comment:
leftover debug? FYI you can have in L237 in case of debugging errors.
EXPECT_TRUE(status.ok()) << status.string() << " " << status.getCode();